static/js/MapView.js

```diff
@@ -64,6 +64,15 @@ MapView = (function($, L, Models, Config) {
         });
         self.tabsControl.addTo(self.map);
 
+        self.sliderControl = L.control.sliderControl({
+            position: "bottomright",
```

Comment:
It doesn't seem to get in the way of the other control if set to `topright`.  Probably it could be laid out better as part of the `TABSControl`, though.